modules/javafx.controls/src/main/java/javafx/scene/control/skin/TreeTableViewSkin.java

```diff
@@ -275,8 +276,6 @@ protected void executeAccessibleAction(AccessibleAction action, Object... parame
         }
     }
 
-
-
```

Comment:
There are no changes in this file other than white space and unused imports, so this file can be reverted.